docs/HowToUseMultiHeap.md

```diff
@@ -1,30 +1,54 @@
 # How to use(add) Multi-heap Regions
 
-There are two cases to use multi-heap regions.  
-1. continuous physical RAM but want to split regions  
+There are two cases to use multi-heap regions.
+1. continuous physical RAM but want to split regions
 2. separated physical RAMs but want to use for heap  
 
-To use multi-heap regions, find below two steps.
+To use multi-heap, find below two steps.
 
-## Set number of Heap regions
+## Set number of Regions
```

Comment:
Need a description of difference between `region` and `heap`

Reply:
done